docusaurus/docs/Android/04-compose/01-overview.mdx

```diff
@@ -32,7 +32,7 @@ See the individual pages for these components to learn more about them.
 * [Message List](04-message-components/03-message-list.mdx)
 * [Message Composer](04-message-components/04-message-composer.mdx)
 * [Attachments Picker](04-message-components/05-attachments-picker.mdx)
-* [Selected Message Overlay](04-message-components/06-selected-message-overlay.mdx)
+* [Selected Message Menu](04-message-components/06-selected-message-menu.mdx)
 
```

Comment:
There should be also [Channel List Item](03-channel-components/04-default-channel-item.mdx) in the channel components above. I've going to add this line in the PR where I will update the docs for `ChannelListItem`. I will also rename the page as it is now named `DefaultChannelListItem`.

Reply:
Yeah I agree!